scripts/sign_image.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash -ex
+
+
+IMAGE=""
+SIGNING_KEY=""
+SIGNING_CERT=""
+CA_CERT=""
+TARGET_PATH=""
+TMP_CERT_PATH="/tmp/cert"
+CERT_PATH=""
+
+usage()
+{
+    echo "Usage:  $0 -i <image_path> [-c <cert_path>] [-t <cert_target_path>]"
+    exit 1
+}
+
+generate_signing_key()
+{
+    SIGNING_CSR="${TMP_CERT_PATH}/signing.csr"
+    CA_KEY="${TMP_CERT_PATH}/ca.key"
+    mkdir -p "${TMP_CERT_PATH}"
+
+    # Generate the CA key and certificate
+    openssl genrsa -out $CA_KEY 4096
+    openssl req -x509 -new -nodes -key $CA_KEY -sha256 -days 3650 -subj "/C=US/ST=Seattle/L=Redmond/O=SONiC/CN=www.sonic.com" -out $CA_CERT
```

Comment:
>"/C=US/ST=Seattle/L=Redmond/O=SONiC/CN=www.sonic.com" [](start = 72, length = 53)

Can we use `"/ST=Test/L=Test/O=Test/CN=Test"` ?

Reply:
Fixed.